### PR TITLE
collect a Map of import and their methods with line numbers

### PR DESCRIPTION
**Pull request by PiyushMishra**

### CODE REVIEW

File: core/src/main/scala/com/betterdocs/parser/MethodVisitor.java [hunk 1]
```diff
@@ -51,6 +51,11 @@
     private HashMap<String, ArrayList<Integer>> lineNumbersMap = new HashMap<String, ArrayList<Integer>>();
     private ArrayList<HashMap<String, ArrayList<Integer>>> listOflineNumbersMap = new
             ArrayList<HashMap<String, ArrayList<Integer>>>();
+    
+    
+	private HashMap<String, ArrayList<Integer>> methodAndLineNumbers = new HashMap<String, ArrayList<Integer>>();
+
```
Comment (ScrapCodes) [inline core/src/main/scala/com/betterdocs/parser/MethodVisitor.java]: fix indentation

File: core/src/main/scala/com/betterdocs/parser/MethodVisitor.java [hunk 2]
```diff
@@ -275,7 +278,24 @@ private void specialVisitBody(BodyDeclaration bdecl, Object arg1) {
 
     @Override
     public void visit(MethodCallExpr n, Object arg) {
-        Expression s = n.getScope();
+		Expression s = n.getScope();
```
Comment (ScrapCodes) [inline core/src/main/scala/com/betterdocs/parser/MethodVisitor.java]: fix indentations here as well.

File: core/src/main/scala/com/betterdocs/parser/MethodVisitor.java [hunk 3]
```diff
@@ -374,6 +394,24 @@ public void setImports(List<ImportDeclaration> imports) {
         }
     }
 
+	private void updateImportWithMethodAndLineNumbers(MethodCallExpr n,
```
Comment (ScrapCodes) [inline core/src/main/scala/com/betterdocs/parser/MethodVisitor.java]: fix indentations here as well.

File: core/src/main/scala/com/betterdocs/parser/MethodVisitor.java [hunk 4]
```diff
@@ -402,4 +440,8 @@ public void visit(CatchClause n, Object arg) {
     public Map<String, String> getImportDeclMap() {
         return importDeclMap;
     }
+    
+	public HashMap<String, HashMap<String, ArrayList<Integer>>> getImportsWithMethodAndLineNumber() {
```
Comment (ScrapCodes) [inline core/src/main/scala/com/betterdocs/parser/MethodVisitor.java]: fix indentations here as well.

**Comment by ScrapCodes** [comment]
Can you fix the formatter of your editor ?

**Comment by PiyushMishra** [comment]
Looks like its fix now.

**Pull request closed without merging**